Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -779,10 +780,13 @@ struct GenericRegionDecodingInputParameters {
 
     AdaptiveTemplatePixel adaptive_template_pixels[12]; // "GBATX" / "GBATY" in spec.
     // FIXME: GBCOLS, GBCOMBOP, COLEXTFLAG
+
+    // If is_modified_modified_read is false, generic_region_decoding_procedure() reads data off this decoder.
+    JBIG2::ArithmeticDecoder* arithmetic_decoder;
```

Comment:
Please initialize to `nullptr`.

In general we should be initializing all POD members in structs/classes. We've had several UB bugs due to uninitialized members.